payment/app.py

```diff
@@ -11,10 +11,13 @@
 database = os.environ['MONGODB_DATABASE']
 gateway_url = os.environ['GATEWAY_URL']
 
-app.config["MONGO_URI"] = f"mongodb://{username}:{password}@{hostname}:27017/{database}"
+# app.config["MONGO_URI"] = f"mongodb://{username}:{password}@{hostname}:27017/{database}"
+app.config["MONGO_URI"] = f"mongodb://{hostname}:27017,{hostname}:27118/{database}"
```

Comment:
I think that the second port should also be 27017 as the command:
```
docker-compose exec router02 mongosh --port 27017
```

works, but 

```
docker-compose exec router02 mongosh --port 27118
```
doesnt. For my changes I had to specify 27017 here as well, whereas my routers where both running on a different port